examples/compiled/circle_binned.vg.json

```diff
@@ -157,7 +157,6 @@
   "legends": [
     {
       "size": "size",
-      "gradientLength": {"signal": "clamp(height, 64, 200)"},
```

Comment:
Could you explain why this is removed?

Reply:
The legend is a symbol legend and doesn't use gradient length. 

You found the code below: https://github.com/vega/vega-lite/pull/6724/files#r453887311